common/utils/EncryptionManager.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const crypto = require('crypto');
+const config = require('../config');
+const CONST = require('../constants');
+
+class EncryptionManager {
+
+  /**
+   * @param text test to be encrypted
+   */
+  encrypt(text) {
+    const cipher = crypto.createCipheriv(CONST.ENCRYPTION.AES_256_ALGORITHM, config.apiserver.encryption.password, config.apiserver.encryption.initialization_vector);
+    let crypted = cipher.update(text, CONST.ENCRYPTION.INPUT_ENCODING, CONST.ENCRYPTION.OUTPUT_ENCODING);
+    crypted += cipher.final(CONST.ENCRYPTION.OUTPUT_ENCODING);
```

Comment:
Some code comments here about LN 14 & 15 will help. This could be std usage from lib functions, but just from a readability part, it would help as LN15 is concat with LN14.

Reply:
actually the final is not required. final is called only when you call update multiple times. I can concat them.

Reply:
actually I can remove the final call.